src/components/cluster/new/cluster_creation_duration.js

```diff
@@ -11,14 +11,18 @@ class ClusterCreationDuration extends React.Component {
 
     if (minutes > 0) {
       return (
-        <p>
+        <p style={{ marginTop: '23px' }}>
```

Comment:
If you're repeating this, why not make another styled component with this property and use it in both cases?

Reply:
Good question: I'm totally fine with inlined styles, maybe it is something we could discuss in SIG Frontend because I think Oliver is not.

The reason why I didn't styled it is because I find it overkill to import `styled` and create a variable just for one css property... but yeah, we can talk about it if you think otherwise.

BTW, I refactored the component, this is how I would have written it. Happy to hear your opinions about it :-)